docs/tutorials/FlowTutorial.md

```diff
@@ -1118,116 +1147,44 @@ global_placement -density 0.6
 
 Read the resulting macro placement with a complete core view:
 
-![Placement Density 60](./images/placement_density_06_full.png)
-
-Zoomed view of cell placement:
-
-![Placement density zoomed](./images/placement_density_06_zoomed.png)
+| <img src="./images/placement_density_06_full.webp" width=450px> | <img src="./images/placement_density_06_zoomed.webp" width=450px> |
+|:--:|:--:|
+| Figure 1: With density 0.6 | Figure 2: Zoomed view of macro and std cell placement |
```

Comment:
Please make sure this will be correctly rendered in the final HTML. You can use the following:
```
cd docs
make html
```
This will create a `build` folder, if you open the `index.html` file in your browser you can navigate the website.
Also, you can check if the links are all working with
```
make checklinks
```

Reply:
`make html` throw following error:
```
[vijayan@chipware-vijayan docs]$ make html
/bin/sh: sphinx-build: command not found
make: *** [html] Error 127
```
Is anything I need to install it?

Reply:
@vijayank88 Not sure if you still need this, but you can see docs/README.md. Best practice is to create a virtualenv/conda env to install the packages from requirements.txt